examples/ui/ui_material.rs

```diff
@@ -87,9 +92,11 @@ fn animate(
         if let Some(material) = materials.get_mut(handle) {
             // rainbow color effect
             let new_color = Color::hsl((time.elapsed_seconds() * 60.0) % 360.0, 1., 0.5);
+            let border_color = Color::hsl((time.elapsed_seconds() * 60.0) % 360.0, 0.75, 0.75);
             material.color = LinearRgba::from(new_color).to_f32_array().into();
             material.slider =
                 ((time.elapsed_seconds() % (duration * 2.0)) - duration).abs() / duration;
+            material.border_color = LinearRgba::from(border_color).to_f32_array().into();
```

Comment:
nit:
```suggestion
            material.border_color = border_color.to_linear().to_vec4();
```

Reply:
ah yeah I thought this looked weird, I'll change the other line as well I copied it from.